src/commands/lambda-unlink-from-application.ts

```diff
@@ -0,0 +1,59 @@
+import {Command} from '@commander-js/extra-typings';
+import {FusionAuthClient, ApplicationRequest} from '@fusionauth/typescript-client';
+import chalk from 'chalk';
+import {errorAndExit, getApplication} from '../utils.js';
+import {apiKeyOption, hostOption} from "../options.js";
+
+const action = async function ( applicationId: string,
+                                lambdaId: string,
+                                {key: apiKey, host}:
+                                    {
+                                        key: string;
+                                        host: string
+                                    }
+                                ): Promise<void>
+{
+    console.log(`Unlinking lambda ${lambdaId} from application ${applicationId} on ${host}`);
+    try {
+        const application = await getApplication(applicationId, {key: apiKey, host});
+        const request: ApplicationRequest = {
+            "application": {
+                "name": application.name,
+                "lambdaConfiguration": {}
+            }
+        };
+        const accessTokenPopulateId = application.lambdaConfiguration?.accessTokenPopulateId;
+        const idTokenPopulateId     = application.lambdaConfiguration?.idTokenPopulateId;
+
+        if (!accessTokenPopulateId && !idTokenPopulateId)
+            errorAndExit(`No existing lambdas were linked`);
+        if (accessTokenPopulateId && accessTokenPopulateId == lambdaId)
+            request.application!.lambdaConfiguration!.accessTokenPopulateId = null;
+        else
+            request.application!.lambdaConfiguration!.accessTokenPopulateId = accessTokenPopulateId;
+        if (idTokenPopulateId && idTokenPopulateId == lambdaId)
+            request.application!.lambdaConfiguration!.idTokenPopulateId = null;
```

Comment:
Could you set it to `undefined` instead of `null`?

`TS2322: Type  null  is not assignable to type  string | undefined `

Reply:
Could you add brackets to this `if` statements? It's hard to read.

Reply:
Setting the properties to `undefined` and updating the application with `patchApplication` works for me

Reply:
setting to an empty string seems to work better to reset the value otherwise using undefined will not actually update the property